src/components/SequenceCard/SequenceCard.js

```diff
@@ -1,30 +1,97 @@
 import React from 'react'
-import styled from 'styled-components'
+import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 
 SequenceCard.propTypes = {
-  description: PropTypes.string,
-  lengthTimeCode: PropTypes.string,
+  description: PropTypes.string.isRequired,
+  lengthTimeCode: PropTypes.string.isRequired,
+  tag: PropTypes.string,
+  timeCodeLowerThirdIn: PropTypes.string,
+  timeCodeLowerThirdOut: PropTypes.string,
+  playerName: PropTypes.string,
 }
 
-export default function SequenceCard({ description, lengthTimeCode }) {
+export default function SequenceCard({
+  description,
+  lengthTimeCode,
+  tag = '',
+  timeCodeLowerThirdIn = '',
+  timeCodeLowerThirdOut = '',
+  playerName = '',
+}) {
   return (
     <Card>
       <Description>{description}</Description>
-      <Timecode>{lengthTimeCode}</Timecode>
+      {tag ? (
+        <InfoIfTag>
+          <TagAndLength>
+            <Tag>{tag}</Tag>
+            <Timecode>{lengthTimeCode}</Timecode>
+          </TagAndLength>
+          <LowerThirdContainer>
+            <Name>
+              Spieler: <span>{playerName}</span>
+            </Name>
+            <TimecodeIn>
+              Bauchbinde IN: <span>{timeCodeLowerThirdIn}</span>
+            </TimecodeIn>
+            <TimecodeOut>
+              Bauchbinde OUT: <span>{timeCodeLowerThirdOut}</span>
+            </TimecodeOut>
+          </LowerThirdContainer>
+        </InfoIfTag>
+      ) : (
+        <Timecode>{lengthTimeCode}</Timecode>
+      )}
     </Card>
   )
 }
 
 const Card = styled.section`
-  border: 1px solid grey;
   border-radius: 10px;
-  padding: 0 10px;
+  border: 1px solid grey;
   margin: 15px 0;
+  padding: 0 10px;
 `
 
 const Description = styled.p``
 const Timecode = styled.p`
   font-weight: 800;
+  margin: 0px;
   text-align: right;
 `
+
+const InfoIfTag = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+const TagAndLength = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+`
+
+const Tag = styled.div`
+  background-color: #e3e3e3;
```

Comment:
In a later step you can think of creating `$variables` for the colors in your `GlobalStyles.js` and then just import them as e.g. `color: var(--main-color);`.

Reply:
I will keep this in mind